core.lua

```diff
@@ -82,7 +105,7 @@ local function executeMessage(input)
         verbose = false
     }
 
-    if(BuffDuty.Console.parseMessageCommand(cmd, LibStub("AceConsole-3.0"):GetArgs(input, 12))) then
+    if(BuffDuty.Console.parseMessageCommand(cmd, LibStub("AceConsole-3.0"):GetArgs(input, 14))) then
```

Comment:
same here

Reply:
I can propose the following format:
```suggestion
    local args = LibStub("AceConsole-3.0"):GetArgs(input, 14)
    if(BuffDuty.Console.parseMessageCommand(cmd, args)) then
```

Also what do you think about having a function like `validateMessageCommand`? Code would look like 
```suggestion
    if(BuffDuty.Console.isCommandValid(args)) then
        cmd = BuffDuty.Console.parseMessageCommand(args)
```

Reply:
It's kind of redundate, as you more or less have to parse the command to validate it... it also makes things more complex and prone to error as now you have to synchronise `isValid` and `parse` to ensure they are checking and reading the same values; much simpler and easier to do it all in one go unless there is a good reason you need to call `isValid` in multiple places and `parse` has an unacceptable performance overhead.

Reply:
Then, parseMessageCommand should raise an error if command is invalid. It's very counter-intuitive that function named "parseMessageCommand" has to return boolean value, not the parsed command.